EthersEvent not defined (typescript)

I'm trying to convert some javascript that uses the Alchemy SDK into TypeScript. It seems the typescript files in the project are malformed, making the library unusable with TypeScript. When I try to import the module, it fails. (This is 1.0.8)

This is my entire test script, it's one line:

`import { initializeAlchemy } from "@alch/alchemy-sdk";`

This results in 
```
node_modules/@alch/alchemy-sdk/dist/src/api/alchemy-websocket-provider.d.ts:3:10 - error TS2305: Module '"../internal/internal-types"' has no exported member 'EthersEvent'.

3 import { EthersEvent } from '../internal/internal-types';
           ~~~~~~~~~~~

node_modules/@alch/alchemy-sdk/dist/src/internal/internal-types.d.ts:10:12 - error TS2304: Cannot find name 'EthersEvent'.

10     event: EthersEvent;
              ~~~~~~~~~~~

node_modules/@alch/alchemy-sdk/dist/src/internal/internal-types.d.ts:45:64 - error TS2304: Cannot find name 'SubscriptionEvent'.

45 export declare type WebSocketMessage = SingleOrBatchResponse | SubscriptionEvent;
                                                                  ~~~~~~~~~~~~~~~~~


Found 3 errors.
```

If I look at the code, it really doesn't have a definition for `EthersEvent`. I'm not a TypeScript expert nor an Alchemy expert, but I don't see how this code could ever work and I'm not sure what to do to go about fixing it.

--- a/src/internal/internal-types.ts
+++ b/src/internal/internal-types.ts
@@ -68,7 +68,7 @@ export type SingleOrBatchResponse = JsonRpcResponse | JsonRpcResponse[];
  * Subscription API types. The original class is not exported by ethers. Minimal
  * changes have been made in order to get TS to compile.
  */
-class Event {
+export class Event {
   readonly listener: Listener;
   readonly once: boolean;
   readonly tag: string;
@@ -136,8 +136,6 @@ class Event {
 /**
  * Wrapper class around the ethers `Event` class in order to add support for
  * Alchemy's custom subscriptions types.
- *
- * @internal
  */
 export class EthersEvent extends Event {
   get address(): string | null {
@@ -153,7 +151,6 @@ export class EthersEvent extends Event {
   }
 }
 
-/** @internal */
 export interface SubscriptionEvent<T = any> {
   jsonrpc: '2.0';
   method: 'eth_subscription';

